eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/batch/io/ClusteringWriter.java

```diff
@@ -79,6 +93,8 @@
 
     private Map<String, Long> assignedAccessions;
 
+    private Map<Long, List<ClusteredVariantEntity>> clusteredVariantsPerAccession;
```

Comment:
What does this do? I can't seem to find where it is being used.

Reply:
This was part of another approach I started with. I just removed this.